lib/Plack/Middleware/AccessLog/Structured.pm

```diff
@@ -36,13 +34,13 @@ sub new {
 sub call {
 	my ($self, $env) = @_;
 
-	my $t_before = Time::HiRes::time();
+	my $t_before = Time::Moment->now;
 	my $res = $self->app->($env);
 
 	return $self->response_cb($res, sub {
 		my ($cb_res) = @_;
 
-		my $t_after = Time::HiRes::time();
+		my $t_after = Time::Moment->now;
```

Comment:
Probably should be `->now_utc` as well, might avoid a conversion when calculating the `request_duration` further down.

Reply:
Done in update